packages/ember-htmlbars/lib/hooks/create-fresh-scope.js

```diff
@@ -46,13 +49,124 @@
   the current view's `controller`.
 */
 
+function Scope(parent) {
+  this._self = null;
```

Comment:
Fixed in https://github.com/rwjblue/ember.js/commit/742cbaa9094361c7e9938bba969cbaf8be778853.